data/virt_autotest/kubevirt_tests/node-helper-patch.yaml

```diff
@@ -6,4 +6,19 @@ spec:
       dnsConfig:
         options:
         - name: ndots
-          value: "1"
\ No newline at end of file
+          value: "1"
+      initContainers:
```

Comment:
Hi Nan, 
is this patch fix from upstream/dev or personally by you? I am wondering, if it is not fix from upstream or dev,  shall we report a bug?

Reply:
The source file (https://gitlab.suse.de/virtualization/kubevirt-ci/-/blob/main/node-helper.yaml.in) is from dev CI, not upstream. Maybe it's not happended in Dev env so far. This patch file is to resolve the specific issue happened in QA env like the existing code,  I think we can just fix this issue until it's updated in dev CI.